### ISSUE TITLE
`ansible_pkg_mgr` is unknown on Travis CI instance after installing homebrew

### ISSUE THREAD

**Issue by GlaIZier** [Comcast/ansible-sdkman#42]
Hello there!

Thank you for your work!

This is a duplicate to an issue described [here](https://github.com/geerlingguy/ansible-role-homebrew/issues/117)
In short, `ansible_pkg_mgr` is unknown after applying [this role](https://github.com/geerlingguy/ansible-role-homebrew) on Travis CI instance.

**Comment by elliotweiser** [comment]
I'll see what I can do to reproduce it.

**Comment by elliotweiser** [comment]
In the meantime...

Since you're running on MacOS and this appears to be a bug that relates to package installation (which was only really tested on Linux distros), I think you can bypass the error by just skipping the `sdkman_privilege` tag. Do you need privilege escalation/root access for testing your playbook on MacOS?

**Comment by GlaIZier** [comment]
Yes, your workaround has worked. However, I'm still not sure that this is a bug of your role.

My pipeline on Travis is as [follows](https://github.com/GlaIZier/mac-dev-environment/blob/master/.travis.yml)
1. remove homebrew
2. run the [role](url) to install homebrew 
3. run this role

I ran some tests. 
Before the first step `ansible_pkg_mgr = homebrew`, after it `ansible_pkg_mgr = unknown` and after the second step it is still `ansible_pkg_mgr = unknown`. 

Seems like homebrew role doesn't set up the `ansible_pkg_mgr` fact... Maybe I could do it manually before executing your role?

**Comment by elliotweiser** [comment]
It looks MacOS support needs some rework. AFAIK, there should be no need to install homebrew packages, so this task can probably just be skipped if `ansible_os_family == "Darwin". We can tackle this next.

Incidentally, I think if you configure your playbook to run the homebrew role and the sdkman role in separate plays, then `gather_facts` will repopulate the `ansible_pkg_mgr` fact with the appropriate value. Granted, I still think there's nothing homebrew-related that should be happening here.

Stay tuned for updates.

**Comment by GlaIZier** [comment]
As a workaround, I set up `ansible_pkg_mgr ` manually in `pre_tasks` since I also use another role and experience the same issue.
